test/CMakeLists.txt

```diff
@@ -77,8 +77,8 @@ endmacro()
 set(COMMON_TEST_FILES
     "manifest"
     "detail/huang_lu"
-    "gate_id"
-    "utils/connectedness_check"
+    #"gate_id"
```

Comment:
Can we re-enable these two tests, or do they fail?

Reply:
They are actually working. I don't know, why they were not in the original list of the tests to run...